### PR TITLE
NLRFIM-115 - Custom URL paths in Java

### PR DESCRIPTION
**Pull request by TauBeblein**

### CODE REVIEW

File: oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java [hunk 1]
```diff
@@ -9,7 +9,13 @@
     // SP configuration
     private String spEntityID; // This SP's EntityID
     private String baseUrl; // The URL endpoint that the DispatcherServlet is working on
-    private boolean validationEnabled = true;
+	private String servletRoutingPathPrefix; // The endpoint prefix that the DispatcherServlet is working on
```
Comment (mdbech) [inline oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java:12]: There is some issues with tabs versus spaces. Could reformat to 4 spaces as the rest?

Comment (mdbech) [reply oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java:12]: The prefix and suffixes must be optional configuration parameters

**Review by mdbech — CHANGES_REQUESTED** [review]
Please also remember the RELEASE_NOTES.md-file here

File: oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java (hunk 1 above)
Comment (TauBeblein) [reply oiosaml/src/main/java/dk/gov/oio/saml/config/Configuration.java]: done

**Review by mdbech — APPROVED** [review]

**Pull request merged**